.changeset/sixty-yaks-stare.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
❓ Is this file required?

Reply:
Yes, a changeset file needs to be committed, but we can leave it empty